### ISSUE TITLE
Update suggestion to README.md

### ISSUE THREAD

**Issue by HotelLima521** [Matoking/protontricks#30]
I added Portage packages for Gentoo

It's a text file being Github doesn't allow me to push markdown files.

Sorry if this is the wrong place to put this suggestion/request at.

[readme.txt](https://github.com/Matoking/protontricks/files/3641239/readme.txt)

**Comment by HotelLima521** [comment]
Also, changes are made on lines 78, and 114. Added these, due to the fact I had to search for the packages a few times just to ensure I had the dependencies, and also added the symbolic link for the path which took me a while to figure out why pipx wasn't working

**Comment by Matoking** [comment]
The file you linked appears empty. You could use [gist.github.com](https://gist.github.com/) if GitHub's in-site editor isn't behaving nicely for some reason.

Also, [Repology](https://repology.org/project/protontricks/versions) doesn't know of any Gentoo version of Protontricks. Is the custom package located on another site that isn't known by Repology?

**Comment by HotelLima521** [comment]
I don't think there is, unless someone has made the package in Layman, which is sort of similar to Arch's AUR. I meant more so the pipx dependencies [Gist](https://gist.github.com/HotelLima521/f07cef9639743e490dfe4d92ff760931)

Sorry, I'm still kinda learning Github haha

**Comment by Matoking** [comment]
Thanks for your suggestions.

I added the installation commands to README.md, although I removed the command adding a symlink from the home directory to `/usr/bin`. The next instruction suggests closing and opening the terminal emulator, so that `pipx` works without requiring system-wide changes.
